shell/src/main/java/alluxio/master/job/JobMasterRpcHealthCheckClient.java

```diff
@@ -25,8 +25,8 @@
 /**
  * JobMasterHealthCheckClient check if job_master is serving RPC.
  */
-public class JobMasterHealthCheckClient extends RpcPortHealthCheckClient {
-  private static final Logger LOG = LoggerFactory.getLogger(JobMasterHealthCheckClient.class);
+public class JobMasterRpcHealthCheckClient extends RpcPortHealthCheckClient {
```

Comment:
Actually, regular health-check clients are not doing RPC health check. Do we even need these separate classes?

Reply:
Sorry, I don't quite understand. Are you saying that all the other health check clients aren't making any RPC's at all?